### ISSUE TITLE
Warnings on Starting Server

### ISSUE THREAD

**Issue by saurabh-mish** [McShelby/hugo-theme-relearn#609]
I setup this theme locally by:

```
hugo new site myproject
cd myproject
git init
git submodule add https://github.com/McShelby/hugo-theme-relearn themes/hugo-theme-relearn
```

On running the server using this theme:

```
cd themes/hugo-theme-relearn/exampleSite
hugo server
```

There are a lot of warnings:

```
Start building sites … 
hugo v0.114.1+extended darwin/arm64 BuildDate=unknown

WARN  "tests/frontmatter.en.md": UNSUPPORTED usage of 'attachments' shortcode found while using Hugo >= 0.112.0, use a leaf bundle instead; see https://mcshelby.github.io/hugo-theme-relearn/shortcodes/attachments/index.html#single-language
WARN  "tests/attach/attach3.en.md": UNSUPPORTED usage of 'attachments' shortcode found while using Hugo >= 0.112.0, use a leaf bundle instead; see https://mcshelby.github.io/hugo-theme-relearn/shortcodes/attachments/index.html#single-language
WARN  "tests/attach/attach1/_index.en.md": UNSUPPORTED usage of 'attachments' shortcode found while using Hugo >= 0.112.0, use a leaf bundle instead; see https://mcshelby.github.io/hugo-theme-relearn/shortcodes/attachments/index.html#single-language
WARN  "tests/lists/_index.en.md": UNSUPPORTED usage of 'attachments' shortcode found while using Hugo >= 0.112.0, use a leaf bundle instead; see https://mcshelby.github.io/hugo-theme-relearn/shortcodes/attachments/index.html#single-language
WARN  "tests/attach/attach3.pir.md": UNSUPPORTED usage of 'attachments' shortcode found while using Hugo >= 0.112.0, use a leaf bundle instead; see https://mcshelby.github.io/hugo-theme-relearn/shortcodes/attachments/index.html#single-language
WARN  "tests/attach/attach1/_index.pir.md": UNSUPPORTED usage of 'attachments' shortcode found while using Hugo >= 0.112.0, use a leaf bundle instead; see https://mcshelby.github.io/hugo-theme-relearn/shortcodes/attachments/index.html#single-language
```

Referring to [the linked URL](https://mcshelby.github.io/hugo-theme-relearn/shortcodes/attachments/index.html#single-language), its not clear to me on how to fix it.

**Comment by McShelby** [comment]
I can not reproduce this with your used Hugo version on Windows.

Please update Hugo to the latest version and try again.

If this still occurs, please check if you are using a pristine theme installation.

**Comment by McShelby** [comment]
The warnings are expected for the example page as it also functions as my test site.

**Comment by saurabh-mish** [comment]
I closed a previous issue and reopened it, editing the title and content.

**Comment by saurabh-mish** [comment]
Can you let me know how I can prevent the warnings locally ? 

The URL doesn't seem clear.

**Comment by McShelby** [comment]
> Can you let me know how I can prevent the warnings locally ?

Don't build the exampleSite.

**Comment by McShelby** [comment]
Fixed in #612
